x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/ResponseValueUtils.java

```diff
@@ -114,7 +114,7 @@ private static Object valueAt(DataType dataType, Block block, int offset, BytesR
             case LONG, COUNTER_LONG -> ((LongBlock) block).getLong(offset);
             case INTEGER, COUNTER_INTEGER -> ((IntBlock) block).getInt(offset);
             case DOUBLE, COUNTER_DOUBLE -> ((DoubleBlock) block).getDouble(offset);
-            case KEYWORD, TEXT -> ((BytesRefBlock) block).getBytesRef(offset, scratch).utf8ToString();
+            case KEYWORD, SEMANTIC_TEXT, TEXT -> ((BytesRefBlock) block).getBytesRef(offset, scratch).utf8ToString();
```

Comment:
Minor suggestion, swap `SEMANTIC_TEXT` and `TEXT`.

Reply:
No problem with changing this - but I am curious what's the reason? Right now they are in alphabetical order.
If someone takes a look at this code a year from now, they might appreciate having these in alphabetical order rather than having `SEMANTIC_TEXT` at the end which specifies that support was added for `SEMANTIC_TEXT` after `KEYWORD` and `TEXT` are available.

Reply:
When I wrote this code originally, and many other similar switches in the compute engine, I deliberately did not follow a strict style consistency across the codebase, but rather preferred a logical and type grouping style. With a preference for more common and narrower types earlier in the cases.
I have no strong preference of ordering of this particular case label here.